datahub-web-react/src/app/shared/tags/TagProfileDrawer.tsx

```diff
@@ -0,0 +1,417 @@
+                                    {updatedDescription && (
+                                        <ExpandedActions>
+                                            {overLimit && (
+                                                <ReadLessText
+                                                    onClick={() => {
+                                                        setExpanded(false);
+                                                    }}
+                                                >
+                                                    Read Less
+                                                </ReadLessText>
+                                            )}
+                                            {EditButton}
+                                        </ExpandedActions>
+                                    )}
+                                </>
+                            ) : (
+                                <>
+                                    <StripMarkdownText
+                                        limit={ABBREVIATED_LIMIT}
+                                        readMore={
+                                            <>
+                                                <Typography.Link
+                                                    onClick={() => {
+                                                        setExpanded(true);
+                                                    }}
+                                                >
+                                                    Read More
+                                                </Typography.Link>
+                                            </>
+                                        }
+                                        suffix={EditButton}
+                                    >
+                                        {updatedDescription}
+                                    </StripMarkdownText>
+                                </>
+                            )}
+                        </>
+                    )}
+                    {/* Edit Description Modal */}
+                    {showEditModal && (
+                        <div>
+                            <EditDescriptionModal
+                                title={updatedDescription ? 'Update description' : 'Add description'}
+                                description={updatedDescription}
+                                onClose={onCloseEditModal}
+                                onSubmit={(value: string) => {
+                                    handleSaveDescription(value.trim());
+                                }}
+                                isAddDesc={!updatedDescription}
+                            />
+                        </div>
+                    )}
+                    <Divider />
+                    {/* Tag Charts, Datasets and Owners */}
+                    <HeaderLayout>
```

Comment:
This isn't really a header since its at the bottom of the pane? Maybe `DetailsLayout`?

Reply:
Fixed and Commited.

[https://github.com/linkedin/datahub/pull/4132/commits/f314d27106cd3135e7def6ac9db381a764eb4d73](url)